### DISCUSSION TITLE
23.12.04 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#81] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 플레이리스트 음악 저장 기능 구현 마무리
  - 알림 기능 테스트 코드 수정중(컨텍스트 초기화 없이 되도록)
- 내일은 어떤 일을 하나요?
  - 테스트 케이스 정리
  - 음악 저장 기능 테스트 작성

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 친구 신청 수락 기능 테스트 작성
- 내일은 어떤 일을 하나요?
    - 친구 조회 기능 테스트 작성

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 최근에 작업을 하지 못했네요..
* 내일은 어떤 일을 하나요?
  * 쿼리 최적화
